EntAppProject/src/main/java/com/jobook/EntAppProject/Services/UserServiceStub.java

```diff
@@ -12,16 +12,17 @@
 	
 	public User login(String userName, String password) {
 		User user = null;
-		
-		if ((userName.trim().equalsIgnoreCase("test") || userName.trim().equalsIgnoreCase("test@test.com")) && password.trim().equals("Password"))
+		String trimmedUserName = userName.trim();
```

Comment:
I extracted the common expressions of `userName.trim()` and `password.trim()` our of the conditionals and into separate variables to reduce margin for error. the variables are then utilized in the conditionals instead so that if down the road the team needs to change the value of that variable, they can do so in one place and not in all of the conditionals.